data/json/item_groups.json

```diff
@@ -9057,7 +9057,8 @@
       ["distaff_spindle", 50],
       ["carding_paddles", 50],
       ["knitting_needles", 50],
-      ["pointy_stick", 50]
+      ["pointy_stick", 50],
+	  ["atlatl", 20]
```

Comment:
probably inserted a tab here.  Also protip for inserting entries into json, insert new entries into a random spot in the list to avoid conflicts.